components/authz-service/testhelpers/testhelpers.go

```diff
@@ -139,15 +139,28 @@ func (tf *TestFramework) Shutdown(t *testing.T, ctx context.Context) {
 // SetupProjectsAndRulesWithDB is a simplified test framework
 // useful for integration tests with just the DB.
 func SetupProjectsAndRulesWithDB(t *testing.T) (
-	api.ProjectsClient, *TestDB, storage.Storage, int64) {
+	api.ProjectsClient, api.PoliciesClient, *TestDB, storage.Storage, int64) {
```

Comment:
Revisions to this setup function are all to make the `polSrv` available since, again, default roles need to exist when creating projects. This provides access to `CreateRole`.